lib/build.js

```diff
@@ -60,7 +62,6 @@ function build (gyp, argv, callback) {
       buildType = config.target_defaults.default_configuration
       arch = config.variables.target_arch
       nodeDir = config.variables.nodedir
-      copyDevLib = config.variables.copy_dev_lib == 'true'
```

Comment:
There is no need to cope node.lib into node_dir/[Debug | Release] directory. See PR #964